projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -35,3 +35,10 @@
     color: var(--tui-text-secondary);
     margin-inline-end: 0.25rem;
 }
+
+.t-search {
+    position: sticky;
+    top: 0.25rem;
+    background: var(--tui-background-elevation-2);
```

Comment:
Let's also add `box-shadow: 0 -1rem var(--tui-background-elevation-2)` to cover the gap above input from seeing items through.